src/components/modals/OperationDetails.js

```diff
@@ -203,6 +232,51 @@ const OperationDetails = connect(mapStateToProps)((props: Props) => {
               )}
             </Box>
           </Box>
+          {subOperations.length > 0 && account.type === 'Account' && (
+            <React.Fragment>
+              <OpDetailsSection>
+                {t('operationDetails.tokenOperations')}
+                <LabelInfoTooltip
+                  text={t('operationDetails.tokenTooltip')}
+                  style={{ marginLeft: 4 }}
+                />
+              </OpDetailsSection>
+              <Box>
+                {subOperations.map((op, i) => {
+                  const opAccount = (account.tokenAccounts || []).find(
+                    acc => acc.id === op.accountId,
+                  )
+
+                  if (!opAccount) return null
+
+                  return (
+                    <Box style={{ marginLeft: -20, marginRight: -20 }}>
+                      <OperationC
+                        compact
+                        text={opAccount.token.name}
+                        operation={op}
+                        account={opAccount}
+                        parentAccount={account}
+                        key={`${account.id}_${operation.id}`}
+                        onOperationClick={() =>
+                          openModal(MODAL_OPERATION_DETAILS, {
+                            operationId: op.id,
+                            accountId: op.accountId,
+                            parentId: account.id,
+                          })
+                        }
+                        t={t}
+                      />
+                      {i < subOperations.length - 1 && <B />}
```

Comment:
What is `<B>` :thinking:

Reply:
A [styled `Bar` component](https://github.com/pbondoer/ledger-live-desktop/blob/223fc2823e8a646b985f79f9ffa0146f0e1a7b64/src/components/modals/OperationDetails.js#L102) apparently. Not sure why but since the rest of the file used it I rolled with it 🅱️